pkg/costmodel/router.go

```diff
@@ -1156,6 +1156,66 @@ func (a *Accesses) GetInstallNamespace(w http.ResponseWriter, r *http.Request, _
 	w.Write([]byte(ns))
 }
 
+type InstallInfo struct {
+	Containers  []ContainerInfo   `json:"containers"`
+	ClusterInfo map[string]string `json:"clusterInfo"`
+}
+
+type ContainerInfo struct {
+	ContainerName string `json:"containerName"`
+	Image         string `json:"image"`
+	StartTime     string `json:"startTime"`
+	Restarts      int32  `json:"restarts"`
+}
+
+func (a *Accesses) GetInstallInfo(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+
+	pods, err := a.KubeClientSet.CoreV1().Pods(env.GetKubecostNamespace()).List(context.Background(), metav1.ListOptions{
```

Comment:
Do we need to filter on pod status as well? Albeit rare, I've run into issues where there can be multiple cost-model pods (where some have a status of `Evicted`). For example:

To automatically run `kubectl logs` against the cost-model pod: 
```bash
$ kubectl get pods -n kubecost -l app=cost-analyzer -o jsonpath='{.items[0].metadata.name}' | xargs -I {} kubectl logs -f -n kubecost -c cost-model {}
```

where the result of: 
```bash
kubectl get pods -n kubecost -l app=cost-analyzer -o jsonpath='{.items[0].metadata.name}'
```
gives me one of the `Evicted` pods. 

It's not obvious whether or not `Limit: 1` via the API is different than just taking the first result.

Reply:
Good question, even if I get back the evicted pod the info I am pulling from the containers and pods should still be relevant enough for the endpoint (high level install info). If we only go after active pods that could cause a problem if the pods haven't passed the readiness probe because they are new or are actually having an issue.

Reply:
And if we get to the point where the container is dying then the UI will be useless as well 🤷

Reply:
I know I do slightly funky things in my dev cluster, but I think I have an example for what Bolt is talking about. I have my Kubecost workloads scheduled on spot nodes. When the spot nodes die/are restarted, the `kubecost-cost-analyzer` pod ends in a "Completed" state. For example, this is my cluster right now:

```
→ k get pod -n kubecost -l app=cost-analyzer 
NAME                                      READY   STATUS      RESTARTS   AGE
kubecost-cost-analyzer-65c7d58669-kc4hz   0/2     Completed   0          16h
kubecost-cost-analyzer-6cc89cdc5-h9wn7    2/2     Running     0          17m
```

The 17m pod has been updated while I do dev work, which I normally do with the following:
```
kubectl set image deployment/kubecost-cost-analyzer cost-model=FOO --namespace kubecost
```

And this is what happens when I get the container images with some JSONPath. Note the differing container images:
```
→ k get pod -n kubecost -l app=cost-analyzer -o jsonpath='{range .items[*]}{.metadata.name}{range .spec.containers[*]}{"\n"}{"\t"}{.name}{"\t"}{.image}{"\n"}'

kubecost-cost-analyzer-65c7d58669-kc4hz
	cost-model	michaelkubecost/opencost:test-01

	cost-analyzer-frontend	gcr.io/kubecost1/frontend:prod-1.94.3

		
kubecost-cost-analyzer-6cc89cdc5-h9wn7
	cost-model	michaelkubecost/cost-model:log-01

	cost-analyzer-frontend	gcr.io/kubecost1/frontend:prod-1.94.3
```

Reply:
Added `FieldSelector: "status.phase=Running",`